### PR TITLE
initial provider creation docs

### PR DESCRIPTION
**Pull request by GabrielBlondeau-Kirby**
The initial provider creation docs.

I'm not very sure `available_on_bank_holidays` what it means so I need you

### CODE REVIEW

**Comment by KristopherJordan** [comment]
https://en.wikipedia.org/wiki/Bank_holiday

File: source/includes/_providers.md [hunk 1]
```diff
@@ -1 +1,309 @@
...
+Host: api.shareactor.io 
+``` 
+
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{  
+   "first_name":"Mari",
+   "email":"johansenemil@gmail.com",
```
Comment (pedropregueiro) [inline source/includes/_providers.md]: let's not have this sort of data here ;). have a look at how we do with the users

File: source/includes/_providers.md [hunk 2]
```diff
@@ -1 +1,309 @@
...
+   },
+   "deleted":false,
+   "roles":[  
+      "provider"
+   ],
+   "default_address":{  
+      "service":"google",
+      "alias":"",
+      "geo":[  
+         59.927152,
```
Comment (pedropregueiro) [inline source/includes/_providers.md]: let's remove this as well

File: source/includes/_providers.md [hunk 3]
```diff
@@ -1 +1,309 @@
...
+default_position | `geoPoint` | The Provider's position by default
+default_address | `address` | The Provider's address by default. _default lambda: Address_
+schedule | `schedule` | The Provider's schedule. _default lambda: Schedule_
+orders | `list` | List of [`Orders`](#orders)
+products | `list` | List of [`Products`](#products) of Provider. _default lambda: []_
+available_on_bank_holidays | `boolean` | _default `false`_
+country | `string` | The Provider's country. _default `'NOR'`_
+roles | `list` | Define the rolls that a Provider can have. _default ['provider']_
+
+## Retrieve an Product
```
Comment (pedropregueiro) [inline source/includes/_providers.md]: Retrieve a Provider

File: source/includes/_providers.md [hunk 4]
```diff
@@ -1 +1,309 @@
...
+
+``` http 
+GET /providers HTTP/1.1
```
Comment (pedropregueiro) [inline source/includes/_providers.md]: GET /providers/provider_id

File: source/includes/_providers.md [hunk 5]
```diff
@@ -1 +1,309 @@
...
+   },
+   "bio":""
+}
+```
+
+Retrieves the provider with a given ID
+
+Arguments | Type | Description
+--------- | ---- | ------
+**providerid** | `string` | ID of the queried Provider
```
Comment (pedropregueiro) [inline source/includes/_providers.md]: provider_id

File: source/includes/_payments.md [hunk 6]
```diff
@@ -90,7 +90,7 @@ Content-Type: application/json
 }
 ```
 
-Attribute | Description
+Arguments | Description
```
Comment (GabrielBlondeau-Kirby) [inline source/includes/_payments.md]: Because an `Attribute` is not an `Argument` ^^
Attribute if for an Object and Argument for method/function, no ??

**Comment by KristopherJordan** [comment]
look through this one more time @GabrielBlondeau-Kirby

**Pull request merged**